internal/log/log.go

```diff
@@ -153,3 +158,9 @@ func (l *Logger) StartPlatformLogger(cfg ConfigProvider) {
 		handler,
 	)
 }
+
+// Log is intended for use by the AWS SDK logger and is necessary for
+// compatiblity with said package. Will log messages at debug level.
+func (l *Logger) Log(v ...interface{}) {
+	l.Debug(fmt.Sprintln(v...))
+}
```

Comment:
@rohanpm, this should suffice, right? If this func is being used, we want debug anyway (it shouldn't be used outside of the AWS package). I've verified that it works as well.

Reply:
It's probably right, hard to say for sure because the aws.Logger interface didn't document how those multiple arguments are meant to be interpreted.  So it's not super clear if they are meant to be used with a function like Sprintln or used in some other way.  But if you tested it and it looks right, it should be right.

Two points now:

- It should probably be Sprint and not Sprintln.  They're the same except Sprintln puts a newline at the end, but you wouldn't normally do that with a logger because the logger itself takes care of it (e.g. we write logger.Debug("some message") and not logger.Debug("some message\n").
- Just a suggestion: would it make sense if we put some field on these messages so we can identify them as coming from the AWS SDK?  Because right now, since we're only getting strings, I guess there's no robust way to identify it.  What I mean is for example: l.WithField("aws", 1).Debug(...).  Then we can know whether or not a message came from aws by setting if the aws field was set.

Reply:
Good suggestions. Thanks.